### DISCUSSION TITLE
GSoC 2024 How to start

### DISCUSSION THREAD

**Discussion by Preetam-Das26** [openvinotoolkit/openvino#22197] [Google Summer of Code]
Hello,
I am Preetam from India.I am a Btech undergrad at NIT Rourkela interested in Python,Django,HTML,C,C++,CSS,JS,JAVA,git,Github,etc. I am interested in this field and have worked on some few projects in these areas involving ML/AI and python backend with django.I look forward to your participation in GSoC 2024.If possible please let me know if how to contribute as a beginner to make this journey smooth. Also to get selected as GSoC project contributor which steps should be taken and how to begin?
Thank you

**Comment by ameyaatmaa** [comment]
For GSoC you should have an unique problem along with its solution in form of an application based project. If you see the projects of previous year winners they all had used flutter framework to create their project, so i suggest you to use flutter framework for app development and you can use figma for designing the UI for the app.

**Comment by prajak002** [comment]
As a final-year CSE student at IEM Kolkata, I'm enthusiastic about contributing to OpenVINO for GSoC. Proficient in Python and C++, with a strong passion for AI, I'm seeking guidance on identifying potential issues and where to initiate my contributions.

**Comment by adrianboguszewski** [comment]
Please check these materials:
https://medium.com/@neeraj_swe/how-to-clear-google-summer-of-code-a-step-by-step-guide-8d63165c73fa
https://www.geeksforgeeks.org/how-to-prepare-for-gsoc-google-summer-of-code/
